safe/gis/expressions.py

```diff
@@ -39,7 +39,7 @@ def inasafe_impact_analysis_layer(field, feature, parent):
 
     index = layer.fieldNameIndex(field)
     if index < 0:
-        return None
+        return index
```

Comment:
Ok, I understood why you are updating this expression ;-)
Yes, I'm trying on my side.

So can we do it on the user side? 
`if(inasafe_impact_analysis_layer('displaced') is None, inasafe_place_value_coefficient(inasafe_impact_analysis_layer('displaced')), if wrong do something here)` ?